modules/odc_con_customizing-a-developer-catalog-or-its-sub-catalogs.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: CONCEPT
+
+[id="odc_con_customizing-a-developer-catalog-or-its-sub-catalogs_{context}"]
+= Developer catalog and sub-catalog customization
+
+As a cluster administrator, you have the ability to organize and manage the Developer catalog or its sub-catalogs by enabling or disabling the sub-catalog types or disabling the entire developer catalog. In some situations, customers do not want their users to have access to the Developer catalog or one or more of its sub-catalogs.
+
+`developerCatalog: types` includes the following parameters that you must define in a snippet to use them in the YAML view:
+
+* `state`: Defines if a list of developer catalog types should be enabled or disabled.
+* `enabled`: Defines a list of developer catalog types (sub-catalogs) that are visible to users.
+* `disabled`: Defines a list of developer catalog types (sub-catalogs) that are not visible to users.
+
+You can enable or disable the following developer catalog types (sub-catalogs) can be enabled or disabled using the YAML view or the form view.
+
+* Builder images
+* Devfiles
+* Helm Charts
+* Operator Backed
+* Samples
+* Templates
+* Event Sinks
+* Event Sources
```

Comment:
In general, if using plain English terms, the second words should not be capitalized, unless they are known terms - I am not sure about Event Sink/Sources, but suspect they should be lower case - definitely Helm chart should be lower, and "Operator Backed" doesn't look right - maybe "Operator-backed"

Alternatively, use the actual values and tick them to indicate they are values:

* `BuilderImage`
* `Devfile`
* `HelmChart`

Reply:
I am not sure if we can change the casing of these terms. They are displayed in the UI and thus i added them in the content as is. @debsmita1 - could you please confirm if we need to change the casing of these terms or do we keep them?

Reply:
In the UI form, we show the labels, and in the yaml we show the catalog ids ( BuilderImage, Template,Devfile, HelmChart, OperatorBackedService, etc). In the UI form , when the user selects a label, the corresponding id is added to the console customization.

Reply:
Fixed as per your original suggestion @gabriel-rh. @debsmita1 has given the nod to follow your advice on Slack chat.

Reply:
This is how it looks in the UI form
<img width="1719" alt="Screenshot 2022-12-06 at 11 51 47 AM" src="https://user-images.githubusercontent.com/22490998/205836888-0d5d58b4-e626-4c78-aa43-e85be3fe1e49.png">
<img width="1719" alt="Screenshot 2022-12-06 at 11 51 51 AM" src="https://user-images.githubusercontent.com/22490998/205836918-cf9d86ca-c9c4-4481-ac69-2000424ac556.png">

 and in the YAML
<img width="1719" alt="Screenshot 2022-12-06 at 11 52 16 AM" src="https://user-images.githubusercontent.com/22490998/205836993-e42862f0-1422-4ec0-b837-d67a8ae4d02b.png">